cosmos/providers/dbt/core/profiles/tests/test_profiles.py

```diff
@@ -205,13 +197,14 @@ def test_create_profile_vars_postgres(airflow_connection: Connection) -> None:
     airflow_connection.host = host
     airflow_connection.login = login
     airflow_connection.password = password
+    airflow_connection.schema = database
```

Comment:
It may be worth adding a link to the Airflow Postgres hook:
https://github.com/apache/airflow/blob/main/airflow/providers/postgres/hooks/postgres.py#L73

And, perhaps, if the user uses newer Airflow versions to set the `database` parameter.